getconfig.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  All Rights Reserved.
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+#
+#  The contents of this software are proprietary and confidential to RidgeRun,
+#  LLC.  No part of this program may be photocopied, reproduced or translated
+#  into another programming language without prior written consent of
+#  RidgeRun, LLC.  The user is free to modify the source code after obtaining
+#  a software license from RidgeRun.  All source code changes must be provided
+#  back to RidgeRun without any encumbrance.
+
+import cv2
+import logging
+import sys
+import yaml
+
+
+class GetConfigYaml:
+    def __init__(self, model_dir):
+        yaml_params = self.read_file(model_dir)
+        self.params = self.parse_params(yaml_params, model_dir)
+
+    def read_file(self, model_dir):
+        try:
+            with open(model_dir + "param.yaml", "r") as file:
+                return yaml.safe_load(file)
+        except Exception as e:
+            logging.error("YAML file not found")
+            sys.exit(1)
+
+    def parse_params(self, yaml_params, model_dir):
+        def params(): return None
+
+        # Get the preprocess parameters
+        params.resize = yaml_params['preprocess']['resize']
```

Comment:
Honestly, this method to get nested value in the dictionary seems to be insecure. You could try to get security to read those values.